setup.py

```diff
@@ -13,7 +13,7 @@
     python_requires=">=3.8",
     install_requires=[
         "packaging",
-        "prettytable>=3.3.0",
+        "prettytable>=3.10.1",
```

Comment:
```suggestion
        "prettytable>=3.10.2",
```

There is a new version since the last 5 hours 🎉